native/cocos/editor-support/spine-wasm/spine-model.cpp

```diff
@@ -1,58 +1,56 @@
 #include "spine-model.h"
 
 SpineModel::SpineModel() {
-    SpineModel::data = new std::vector<uint32_t>(6, 0);
+    _data.setSize(6, 0);
```

Comment:
Avoid to use `std::vector`, use `spine::Vector` instead which will make binary smaller.